appveyor.yml

```
@@ -8,7 +8,6 @@ environment:
 
 cache:
   - .eslintcache
-  - node_modules
   - '%USERPROFILE%\.electron'
   - '%LOCALAPPDATA%\Yarn'
```

Related to the previous ppoint about where `yarn` uses for a local cache:

```suggestion
   - '%LOCALAPPDATA%\Yarn\Cache\v4'
```

---

On my Win10 machine I see the `v2` folder has ~800MB of packages, so I think making this explicit will help speed up our Windows builds